### PR TITLE
[improvement](statistics)Return -1 to neredis if report olap table row count for new table is not done for all tablets.

### PR DESCRIPTION
**Pull request by Jibing-Li**
Return -1 to neredis if report olap table row count for new table is not done for all tablets.
After this change, nereids could know new table is empty or not. When it's not empty but not reported yet, return -1 as row count to nereids.

### CODE REVIEW

**Comment by Jibing-Li** [comment]
run buildall

**Comment by Jibing-Li** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/analysis/ShowTableStatsStmt.java [hunk 1]
```diff
@@ -75,8 +75,10 @@ public class ShowTableStatsStmt extends ShowStmt implements NotFallbackInParser
             new ImmutableList.Builder<String>()
             .add("table_name")
             .add("index_name")
-            .add("row_count")
+            .add("analyze_row_count")
             .add("update_time")
+            .add("report_row_count")
+            .add("report_row_count_for_nereids")
```
Comment (englefly) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/ShowTableStatsStmt.java:81]: report_row_count_for_nereids: what is the meaning of this row count?

File: fe/fe-core/src/main/java/org/apache/doris/analysis/ShowTableStatsStmt.java [hunk 2]
```diff
@@ -75,8 +75,10 @@ public class ShowTableStatsStmt extends ShowStmt implements NotFallbackInParser
             new ImmutableList.Builder<String>()
             .add("table_name")
             .add("index_name")
-            .add("row_count")
+            .add("analyze_row_count")
             .add("update_time")
```
Comment (englefly) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/ShowTableStatsStmt.java:79]: update_time:
what the meaning of this time, the update time of analyze_row_count or report_row_count?

**Review by morrySnow — DISMISSED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/analysis/ShowTableStatsStmt.java (hunk 2 above)
Comment (Jibing-Li) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/ShowTableStatsStmt.java:79]: The time we collect the row count. This is used for the 4 minutes check to decide use analyze row count or report row count.

File: fe/fe-core/src/main/java/org/apache/doris/analysis/ShowTableStatsStmt.java (hunk 1 above)
Comment (Jibing-Li) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/ShowTableStatsStmt.java:81]: report_row_count_for_nereids return -1 when part of the tablets are reported, report_row_count always return the current value of reported row count, no matter it's partially reported or not.

**Comment by Jibing-Li** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/analysis/ShowTableStatsStmt.java (hunk 2 above)
Comment (englefly) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/ShowTableStatsStmt.java:79]: so, it is better to rename to "last_analyze_time"

File: fe/fe-core/src/main/java/org/apache/doris/analysis/ShowTableStatsStmt.java (hunk 1 above)
Comment (englefly) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/ShowTableStatsStmt.java:81]: in this case, report_row_count should be "-1"

File: fe/fe-core/src/main/java/org/apache/doris/analysis/ShowTableStatsStmt.java (hunk 2 above)
Comment (Jibing-Li) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/ShowTableStatsStmt.java]: removed

**Comment by Jibing-Li** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Comment by Jibing-Li** [comment]
run performance

**Comment by Jibing-Li** [comment]
run p0

**Comment by Jibing-Li** [comment]
run performance

**Review by englefly — APPROVED** [review]

**Pull request merged**